ts/private/ts_project.bzl

```diff
@@ -44,6 +45,7 @@ def _ts_project_impl(ctx):
         execution_requirements["supports-workers"] = "1"
         execution_requirements["worker-key-mnemonic"] = "TsProject"
         progress_prefix = "Compiling TypeScript project (worker mode)"
```

Comment:
Seems wrong to hard-code "(worker mode)" since a worker will only be used if the a worker spawn strategy is used. All we're saying here is that the rule support workers.

Reply:
Agreed.